src/EditorFeatures/CSharpTest/CodeActions/SyncNamespace/SyncNamespaceTests_ChangeNamespace.cs

```diff
@@ -542,7 +544,7 @@ interface Interface1
         void M1(Interface1 c1);   
     }}
 }}</Document>
-<Document Folders=""{documentPath2.folder}"" FilePath=""{documentPath2.filePath}""> 
+<Document Folders=""{documentPath2.folder}"" FilePath=""{documentPath2.filePath}"" CanApplyChange=""false""> 
```

Comment:
This is unintended, will revert.

By the way, it doesn't affect test result because by default we only check new solution obtained from refactoring, not the one from workspace after changes being applied.